databricks/koalas/frame.py

```diff
@@ -1963,6 +1963,68 @@ def sort_values(self, by: Union[str, List[str]], ascending: Union[bool, List[boo
         else:
             return kdf
 
+    def sort_index(self, axis: int = 0, level: int = None, ascending: bool = True,
+                   inplace: bool = False, kind: str = None, na_position: str = 'last') \
+            -> Optional['DataFrame']:
+        """
+        Sort object by labels (along an axis)
+
+        Parameters
+        ----------
+        axis : index, columns to direct sorting
```

Comment:
I'd put that axis 1 is not supported.

Reply:
on that note, can you add a function in `utils.py` called `check_axis` for example that throws an exception if the axis is not 0? It will help us track everywhere where we add it eventually.

Reply:
I've extended the docstrings with 0bf4542. The `check_axis` method sounds like a good idea. However, I will add it with a new PR to keep this one clean, if that's okay for you @thunterdb.